config/samples/core_v1beta1_openstackcontrolplane.yaml

```diff
@@ -87,3 +87,9 @@ spec:
   nova:
     template:
       secret: osp-secret
+      apiServiceTemplate:
+        containerImage: quay.io/tripleozedcentos9/openstack-nova-api:current-tripleo
+      schedulerServiceTemplate:
+        containerImage: quay.io/tripleozedcentos9/openstack-nova-scheduler:current-tripleo
+      conductorServiceTemplate:
+        containerImage: quay.io/tripleozedcentos9/openstack-nova-conductor:current-tripleo
```

Comment:
This will not work as nova has a more nested structure due to cells.

```yaml
$ oc get -o yaml OpenStackControlPlane/openstack | yq -y '.spec.nova'
enabled: true
template:
  apiDatabaseInstance: openstack
  apiDatabaseUser: nova_api
  apiMessageBusInstance: rabbitmq
  apiServiceTemplate:
    containerImage: quay.io/tripleozedcentos9/openstack-nova-api:current-tripleo
    replicas: 1
  cellTemplates:
    cell0:
      cellDatabaseInstance: openstack
      cellDatabaseUser: nova_cell0
      cellMessageBusInstance: unused
      conductorServiceTemplate:
        containerImage: quay.io/tripleozedcentos9/openstack-nova-conductor:current-tripleo
        replicas: 1
      hasAPIAccess: true
  keystoneInstance: keystone
  passwordSelectors:
    apiDatabase: NovaAPIDatabasePassword
    cellDatabase: NovaCell0DatabasePassword
    service: NovaPassword
  schedulerServiceTemplate:
    containerImage: quay.io/tripleozedcentos9/openstack-nova-scheduler:current-tripleo
    replicas: 1
  secret: osp-secret
  serviceUser: nova
```

Reply:
I have also added the nova cells config related to container image in comments so that one can easily refer that and replace it.